### PR TITLE
Minor change to avoid confusion in API Docs for CreateAppV2 endpoint

### PR DESCRIPTION
**Pull request by aaronkanzer**
Bumps example in `CreateAppV2` endpoint to be more appropriate for current use cases.

See https://broadinstitute.slack.com/archives/C02GJ782BAA/p1670361183668239?thread_ts=1670350497.658939&cid=C02GJ782BAA as an example

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green (you can re-run automation tests with a comment saying `jenkins retest`
- [ ] Run the automation tests multiple times in parallel to weed out instability if applicable via a comment saying `multi-test`
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by jdcanas — APPROVED** [review]
Looks good, likely was copy pasted. Thanks! It might fail automation tests for unrelated reasons, in which case I'll force-push it for you...

File: http/src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -2550,9 +2550,7 @@ components:
           schema:
             $ref: "#/components/schemas/CreateAppRequest"
           example:
-            diskConfig:
```
Comment (Qi77Qi) [inline http/src/main/resources/swagger/api-docs.yaml:2553]: I think we're reusing the same example for both v1 and v2 api...you'll need to create a separate model for v2 api if you'd like to update the example

**Review by Qi77Qi — CHANGES_REQUESTED** [review]
I think we're reusing the same example for both v1 and v2 api...you'll need to create a separate model for v2 api if you'd like to update the example

**Comment by jdcanas** [comment]
> I think we're reusing the same example for both v1 and v2 api...you'll need to create a separate model for v2 api if you'd like to update the example

Ah yeah, you'll need to create a copy of this https://github.com/DataBiosphere/leonardo/blob/167242c2d1f7467ee358fc65b0d1629c0666abcd/http/src/main/resources/swagger/api-docs.yaml#L3554

**Comment by aaronkanzer** [comment]
@jdcanas @Qi77Qi Updated here! Let me know when you have the chance to re-review -- thanks!

**Review by Qi77Qi — APPROVED** [review]

**Comment by Qi77Qi** [comment]
thanks @aaronkanzer ....sry abt the delay

**Pull request closed without merging**